core/sections/clause_7_core.adoc

```diff
@@ -504,12 +504,151 @@ NOTE: This schema can also be obtained from https://raw.githubusercontent.com/op
 [[sc_process_inputs]]
 ===== Process inputs
 
-*_Overview:_*
+====== Overview
 
 Each process input is a name/value pair that appears in the `inputs` section of
-an execute request.
+an execute request as show in the following example:
+
+```
+"inputs": {
+   "inputName1": <input value 1>,
+   "inputName2": <input value 2>,
+   etc.
+}
+```
+
+The actual name of each input is its identifier as specified by the input's definition in the <<sc_process_description,process description>>.
+
+====== Cardinality
+
+NOTE: The `maxOccurs` parameter is deprecated in this standard and will be removed in a subsequent version of this document.  See <<deprecated-maxOccurs>> for details.
+
+The cardinality on an input is specified using the `minOccurs` and `maxOccurs` parameters from the input's definition in the <<sc_process_description,process description>>.  The default values of `minOccurs` and `maxOccurs` are 1 indicating that a single input of the corresponding name must be specified in an execute request.  The following table covers the various combinations of `minOccurs` and `maxOccurs` values.
+
+[[cardinality_rules]]
+.Cardinality rules for process inputs.
+[cols="10,10,60,40",options="header"]
+|===
+|minOccurs |maxOccurs |Interpretation |Example
+|0 |0 |Not Allowed |
+|0 |1 |A single input value *may* be optionally specified. | "input": {value}
+|1 |1 |A single input value *must* be specified. | "input": {value}
+|1 |N |At least 1 input value *must* be specified.  +
+Regardless of the number of input values provided, those values *must* be encoded using an array. | "input": [{value}] OR +
+"input": [{value1},...,{valueN}]
```

Comment:
This is not backward compatible (the value cannot be provided directly without array nesting). Is it intentional?
If so, my previous comment about "missing single item (non-array) representation" can be discarded.

Reply:
@fmigneault I'm trying to find out how this is not  backward compatible.  I see Requirement 19 in the current, published, version which seems to say the same thing as lines 536 to 538 ... no?

Reply:
It was only problematic to disambiguate between previous (v1) and the original proposition to remove min/max-occurs assuming the default to omit them would not align with explicitly omitting them since they would not be supported. 
Since the decision is to keep them, then this is not an issue anymore.

Reply:
Closing as per @fmigneault last comment.